README.md

```diff
@@ -551,14 +551,14 @@ Backends are specified in the `backend` configuration block under `providers`.
 
 ```hocon
 backend {
-  default = "Local"
+  allowed = ["Local"]
```

Comment:
Pure ToL, the first one being the default struck me as non-intuitive. For whatever reason it seems less odd to me if it was called `enabled` or `active`... (shrug)